projects/plugins/jetpack/_inc/client/newsletter/email-settings.jsx

```diff
@@ -135,7 +199,123 @@ const EmailSettings = props => {
 					onChange={ handleEnableFeaturedImageInEmailToggleChange }
 				/>
 			</SettingsGroup>
+			{ isBylineEnabled && (
+				<SettingsGroup
+					hasChild
+					disableInOfflineMode
+					disableInSiteConnectionMode
+					module={ subscriptionsModule }
+					className="newsletter-group"
+				>
+					<FormLegend className="jp-form-label-wide">
+						{ __( 'Email Byline', 'jetpack' ) }
```

Comment:
The word "byline" is lowercase in designs.

```suggestion
						{ __( 'Email byline', 'jetpack' ) }
```

Reply:
Thanks for the fix!